tools/test_configs/__init__.py

```diff
@@ -10,7 +11,10 @@ def get_valid_configs(target_name):
     if target_name in TARGET_CONFIGS:
         target_config = TARGET_CONFIGS[target_name]
     else:
-        return {}
+        if 'LWIP' in TARGET_MAP[target_name].features:
```

Comment:
Can this be an `elif` instead? that would make the code a smidgen easier to follow.